extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -81,9 +89,15 @@ export class AmpSlideScroll extends BaseCarousel {
     /** @private @const {boolean} */
     this.snappingInProgress_ = false;
 
-    /** @private {?number}*/
+    /** @private {?number} */
     this.scrollTimeout_ = null;
 
+    /** @private {number} */
+    // 0 - not in an elastic state.
+    // -1 - elastic scrolling (back) to the left of scrollLeft 0.
+    // 1 - elastic scrolling (fwd) to the right of the max scrollLeft possible.
+    this.elasticScrollState_ = 0;
```

Comment:
Generally with directional values I prefer 0/1/-1 - the enum or const names do not explain it any better :)